dg/media/assets/styles/screen.css

```diff
@@ -2160,7 +2160,20 @@ Layout Styles > Vertical Rhythm (vr) spacing between sections & components
 .filterOptions-star {
     padding-left: 20px;
 }
-
+.get-connector {
+    padding-left: 20px;
+    border: 1px solid #FFD700;
+    margin-right: 60px;
+}
+.modal{
```

Comment:
Don't use generic classname modal, this could affect the other modals in the project. Change it to something specific to your own implementation